src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/__tests__/DownloadedPackagesSQLLiteStoreTests.swift

```diff
@@ -20,7 +20,308 @@ import FMDB
 import XCTest
 
 final class DownloadedPackagesSQLLiteStoreTests: XCTestCase {
-	private var store: DownloadedPackagesSQLLiteStore = .inMemory()
+	#if INTEROP
+
+	private var store: DownloadedPackagesSQLLiteStoreV1 = .inMemory()
+
+	override func setUp() {
+		super.setUp()
+		store.close()
```

Comment:
I would expect this in `tearDown()`. Is there any reason why this is here?

Reply:
Moved it to teardown.